jupiter-tests/src/test/java/org/junit/jupiter/engine/extension/TestInstanceFactoryTests.java

```diff
@@ -737,6 +737,7 @@ void test2() {
 
 	private static abstract class AbstractTestInstanceFactory implements TestInstanceFactory {
 
+		@SuppressWarnings("deprecation")
```

Comment:
We should not suppress these warnings but change them to use `AutoCloseable` instead.